CMS/static/scss/global.scss

```diff
@@ -1,7 +1,7 @@
 @import './partials/nav';
 @import './partials/footer';
 @import './partials/pagination';
-@import './partials/forms';
+//@import './partials/forms';
```

Comment:
Can we remove this comment?

Reply:
Next PR will uncomment this and update the imported file.